tests/tt_eager/ops/test_bcast_op.cpp

```diff
@@ -6,7 +6,7 @@
 #include "ttnn/tensor/tensor.hpp"
 #include "ttnn/operations/data_movement/bcast/bcast.hpp"
 #include "common/constants.hpp"
-#include "third_party/magic_enum/magic_enum.hpp"
+#include "magic_enum.hpp"
```

Comment:
Here and everywhere else, we should use `#include <magic_enum.hpp>`, since it's a third-party dependency.

Reply:
OK boss